src/Symfony/Component/Config/Definition/Builder/ExprBuilder.php

```diff
@@ -98,6 +98,18 @@ public function ifNull()
     }
 
     /**
+     * Tests if the value is empty.
+     *
+     * @return ExprBuilder
+     */
+    public function ifEmpty()
+    {
+        $this->ifPart = function ($v) { return empty($v); };
```

Comment:
having the same behavior than ``empty($v)`` is fine with me. If we do something else, we should change the name away from ``ifEmpty`` to avoid WTF moments IMO